### PR TITLE
Backup scripts updates

### PR DESCRIPTION
**Pull request by avolkov**
I managed to replicate backup on my machine and separate repository, I'm also adding the following updates:

* README.md with setup instructions
* Added restore script for gtalug.org website.
* Added `backup-installed-packages.sh` that saves installed packages list 
* Added mailman-subscribers.py dependency file
* Fixed GPG multi-recipient encryption command

### CODE REVIEW

**Comment by avolkov** [comment]
Hey guys, could you review my pull request?

File: scripts/archive-subscribers.sh [hunk 1]
```diff
@@ -6,25 +7,31 @@ git fetch origin
 git pull origin master
 
 # Set up password for Mailman
-MAILMANPASSDIR=${MAILMANPASSFILE:-"${HOME}/GitConfig/InitFiles/GTALUG/"}
```
Comment (myles) [inline scripts/archive-subscribers.sh]: Why?

File: scripts/backup-aliases.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ cd ${ARCHIVEHOME}
 git fetch origin
 git pull origin master
 
-scp -r chris@penguin.gtalug.org:/etc/aliases ${ARCHIVEHOME}
+scp -r ${USER}@penguin.gtalug.org:/etc/aliases ${ARCHIVEHOME}
```
Comment (myles) [inline scripts/backup-aliases.sh]: This should just be a backup user instead of an actual user.

File: scripts/archive-subscribers.sh [hunk 3]
```diff
@@ -6,25 +7,31 @@ git fetch origin
...
+MAILMANPASSDIR=${MAILMANPASSFILE:-${HOME}/GitConfig/InitFiles/GTALUG/}
```
Comment (myles) [inline scripts/archive-subscribers.sh]: Why?

Comment (avolkov) [inline scripts/archive-subscribers.sh]: The bash doc I was looking at didn't have quotes and functionally they make no difference.

File: scripts/backup-aliases.sh (hunk 2 above)
Comment (avolkov) [inline scripts/backup-aliases.sh]: Yes, maybe in the next pull request, but for now more generic code is better than less generic code.

**Pull request merged**